index.html

```diff
@@ -18,28 +18,21 @@
     <header id="top">
          <nav>    
             <ul>
-            <div class="box">
                <li><a href="index.html">Home</a></li>
                <li><a href="links.html">Link</a></li>
                <li><a href="glossary.html">Glossary</a></li> 
-             </div>
             </ul>
         </nav>
-    <div class="back-to-top"><a href="◤"" >
-    </div>
     </header>
    
     <main>
     
         <section class="bio">
             <h1 class="intro">Welcome to YAOTIAN TONG’s CSD120 Website</h1>
-            <div paperclip>
             <figure class ="mugshot">
+                <div class="paperclip"></div>
                 <a href="stylesheet"><img src="stylesheet/me.jpg" alt="me" width="80" height="80" ></a>
-                <div >
-                <figcaption style="float:right">A picture of me</figcaption>
-                </div>
-            </div>
+                <figcaption>Me</figcaption>
```

Comment:
I changed your caption to just 'Me' because your thumbnail is so small.  Having a caption that is wider than your thumbnail image causes the paperclip image to appear misaligned because it is positioned relative to the `<figure>` element